src/post/publishedDate.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'; 
+
+class PublishedDate extends React.Component{
+    render(){
+        return(
+            <p>{this.props.publishedDate}</p>
+        )
+    }
+}
```

Comment:
Need to add the code below for using other components
"export default PublishedDate;"

Reply:
Fix plz 😢